dispatchr-ruby/app/models/item.rb

```diff
@@ -1,4 +1,4 @@
 class Item < ApplicationRecord
   has_many :stores, through: :item_stores
-  has_many :request_items, through: :request_items
+
```

Comment:
I think it should just be has_many :request_items. Otherwise we can't access the item's associated request items.